mmdet/models/roi_heads/roi_extractors/groie.py

```diff
@@ -0,0 +1,151 @@
+"""Generic RoI Extractor.
+
+A novel Region of Interest Extraction Layer for Instance Segmentation.
+"""
+
+from __future__ import division
+
+from mmcv.cnn import ConvModule
+from torch import nn
+
+from mmdet.core import force_fp32
+from mmdet.models.builder import ROI_EXTRACTORS
+from mmdet.ops import GeneralizedAttention, NonLocal2D
+from .single_level import SingleRoIExtractor
+
+
+class NoProcess(object):
+    """Apply identity function."""
+
+    def __init__(self, *args, **kwargs):
+        pass
+
+    def __call__(self, x):
+        return x
+
+
+def SequentialModule(modules):
+    """Apply a sequential module."""
+    nn_modules = []
+    for mc in modules:
+        type_ = mc.pop('type')
+        nn_modules.append(load_processing_model(type_, mc))
+    return nn.Sequential(*nn_modules)
+
+
+# Models supported by GRoIE
+models = {
+    'NonLocal2D': NonLocal2D,
+    'GeneralizedAttention': GeneralizedAttention,
+    'ConvModule': ConvModule,
+    'Sequential': SequentialModule,
+    'ReLU': nn.ReLU,
+}
+
+
+def load_processing_model(model_type, config):
+    """Load a specific module."""
+    return models.get(model_type, NoProcess)(**config)
```

Comment:
`NoProcess` might not be necessary, `nn.Sequential()` could do the same thing?

Reply:
I added it inside the plugin list instead of use `NoProcess` class :+1: